### PR TITLE
feat(frontend): add BBB iframe

### LINKED ISSUE #947: 🚀 [Feature] Frontend: Use IFrame for BBB window
## 🚀 Feature
As user I want to stay on the dreammall app to see the Meeting (Table/MallTalk)
See https://www.bigbluemeeting.com/docs/iframe

## Waiting for
- [ ] #946
- [ ] Design

## 🤖 ToDo
- [ ] Add iframe for BBB

### PR DESCRIPTION
**Pull request by Bettelstab**
## 🍰 Pullrequest
Adds a component which opens the user's room at bigbluebutton in an iframe, and a route where this component is used.
Screenshot 2024-06-11 at 11 24 30
Not functional on localhost, supposedly because of CSP / x-frame-options of BBB.

### Issues
- fixes #947 (at least to some basic degree)

### Todo
- [x] Deploy on stage1 to see if it works
- [x] Tests
- [ ] Stories

Remark: If we could have a BBB test instance which allowed for iframes hosted by localhost, that would be awesome for future development.

### CODE REVIEW

File: frontend/src/components/embedded-room/useMyRoom.test.ts [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    })
+
+    it('calls the API', () => {
+      expect(joinMyRoomQueryMock).toBeCalled()
+    })
+
+    it('shows iframe with correct url', () => {
+      expect(wrapper.find('iframe').exists()).toBe(true)
+      expect(wrapper.find('iframe').attributes('src')).toBe(testUrl)
+    })
```
Comment (Elweyn) [inline frontend/src/components/embedded-room/useMyRoom.test.ts:46]: This seems not to be defined we are in useMyRoom that only defines the query.

**Review by Elweyn — CHANGES_REQUESTED** [review]
For the snapshots changes: ``npm run test:unit:update`` and for the error in useMyRoom.test.ts the iframe component is not imported so I think it is right that it is not present.

**Comment by Bettelstab** [comment]
Please have another look @Elweyn 
The ApolloWrapper for Storybook is still not working unfortunately.

**Comment by Bettelstab** [comment]
Regarding Apollo mocking, I suggest to merge this PR, create an issue and wait for https://github.com/lifeiscontent/storybook-addon-apollo-client/pull/83 which hopefully will add Vue support soon to that addon.

**Review by Elweyn — APPROVED** [review]

**Pull request merged**